mimic/model/clb_objects.py

```diff
@@ -38,6 +42,28 @@ def __init__(self):
         self.lbs = {}
         self.meta = {}
 
+    def __len__(self):
+        """
+        The number of cloud load balancers in this collection.
+        """
+        return len(self.lbs)
+
+    def __getitem__(self, clb_id):
+        """
+        Returns the cloud load balancer identified by clb_id.
+        """
+        return self.lbs[clb_id]
+
+    def __contains__(self, clb_id):
+        """
+        Returns true if the CLB ID is registered with our list of load
+        balancers.
+        """
+        for i in self.lbs:
```

Comment:
This is a particularly inefficient and verbose way of implementing this check.  (If efficiency were the only concern I almost certainly wouldn't care, but it also makes the code less clear).

`self.lbs` is a dictionary, and therefore already has a `__contains__` method which checks for key presence using hashing.  So you could implement this to simply do `return (clb_id in self.lbs)`, or even just `return self.lbs.__contains__(clb_id)`.